package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "shelljs",
-  "version": "0.3.0",
+  "version": "0.3.1",
```

Comment:
Actually I think you're right, we should go with minor. I was originally thinking patch since relatively few people are using node 0.12 or iojs, but minor makes more sense in a semver-ish kinda way. Will update this.